src/main/java/fr/romitou/mongosk/skript/expressions/documents/ExprDocumentValue.java

```diff
@@ -94,19 +97,21 @@ public void change(Event e, Object[] delta, Changer.ChangeMode mode) {
                 document.put(value, null);
                 break;
             case ADD:
-                List<Object> addList = document.getList(value, Object.class);
-                if (addList == null) {
-                    document.put(value, deltaList);
-                    return;
+                try {
+                    ArrayList<Object> addList = new ArrayList<>(document.getList(value, Object.class));
```

Comment:
as stated in my first comment, i think we should check if the list is not null before create an arraylist
since `addList` list can not be in the document, we can simply make a `Document#put` to create the list
```suggestion
                    List<Object> addList = document.getList(value, Object.class);
                    if (addList == null) {
                        document.put(value, deltaList);
                        return;
                    }
                    ArrayList<Object> addList = new ArrayList<>(addList);
```

Reply:
If there is any null somewhere, the catch will be triggered, so needn't to check if it's null or not.
Nice on for the ``Document#put`` 🦄 (my editor told me it was a dead code, I didn't check my bad)

Reply:
the problem here is if the addList is null, nothing will happen due to the try/catch
however, if the list is null, so if the list doesn't exist in the document, we must create it